platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/network/connectivity/SocketFactoryTest.java

```diff
@@ -162,6 +164,44 @@ void tlsFactoryTest(final AddressBook addressBook, final Map<NodeId, KeysAndCert
                 NetworkUtils.createSocketFactory(node2, addressBook, keysAndCerts2, TLS_IP_TOS_CONFIG));
     }
 
+    @ParameterizedTest
+    @MethodSource({"com.swirlds.platform.crypto.CryptoArgsProvider#basicTestArgs"})
+    void tlsFactoryTestDynamicPeers(final AddressBook addressBook, final Map<NodeId, KeysAndCerts> keysAndCerts)
+            throws Throwable {
+        assertTrue(addressBook.getSize() > 1, "Address book must contain at least 2 nodes");
+        // choose 2 random nodes to test
+        final Random random = new Random();
```

Comment:
Random seed should be printed, so failing test can be reproduced

```
        final Random random = RandomUtils.getRandomPrintSeed();
```

Reply:
Nice, thank you.